charts/kong/README.md

```diff
@@ -201,6 +205,45 @@ The package to run can be changed via `image.repository` and `image.tag`
 parameters. If you would like to run the Enterprise package, please read
 the [Kong Enterprise Parameters](#kong-enterprise-parameters) section.
 
+### Separate admin and proxy nodes
+
+Users may wish to split their Kong deployment into multiple instances that only
+run some of Kong's services, e.g. where some nodes only run the proxy and other
+only run the admin API, or where some nodes only run Developer Portal services.
```

Comment:
or, in case of Kong Enterprise, some nodes only run Developer Portal services.

Reply:
IMO this is already self-evident: a node that can run Portal services is necessarily Enterprise. Adding the clarification feels like it makes the sentence more awkward, so I've left this as-is.